docs/technical_reference/unit_models/zero_order_unit_models/feed_zo.rst

```diff
@@ -13,6 +14,24 @@ Costing Method
 --------------
 This unit does not include costing.
 
+Additional Variables
+--------------------
+
+.. csv-table::
+   :header: "Description", "Variable Name", "Units"
+
+   "Volumetric flowrate in feed", "flow_vol", ":math:`m^3/s`"
+   "Component mass concentrations", "conc_mass_comp", ":math:`kg/m^3`"
+
+Additional Constraints
+----------------------
```

Comment:
For the other unit models, additional variables and constraints section were written if they were not already covered by the "model type" helper functions (e.g., SIDO, PT, etc.), which have their own documentation. However in this case, FeedZO doesn't use any of those helper functions and inherits from IDAES' `Feed` block class. Essentially, FeedZO add volumetric flowrate and mass concentration as _variables_, in connection with the zero-order property model's state variable that is mass flow rate.

Something that makes this useful is that the user could enter their feed flow rate (volumetric) and concentrations without having to figure out mass flowrates of each component in the feed, which could be less direct. We probably want to elaborate on that in the first section.